examples/advanced/llm_hf/README.md

```diff
@@ -0,0 +1,131 @@
+# LLM Tuning via HuggingFace SFT Trainer
+This example shows how to use [NVIDIA FLARE](https://nvidia.github.io/NVFlare) for Large Language Models (LLMs) tuning tasks. It illustrates how to adapt a local training script with [HuggingFace](https://huggingface.co/) trainer to NVFlare.
+
+## Introduction 
+This example illustrates both supervised fine-tuning (SFT) and parameter-efficient fine-tuning (PEFT) using the [SFT Trainer](https://huggingface.co/docs/trl/sft_trainer) from [HuggingFace](https://huggingface.co/).
+We used the [Llama-2-7b model](https://huggingface.co/meta-llama/Llama-2-7b) to showcase the functionality of federated SFT and PEFT, allowing HuggingFace models to be trained and adapted with NVFlare. 
+Mainly on two fronts:
+- Adapt local HF training scripts to federated application
```

Comment:
I think we can just spell out HuggingFace here

Reply:
sounds good, changed